common/src/main/java/com/flipkart/varadhi/exceptions/NotInitializedException.java

```diff
@@ -0,0 +1,7 @@
+package com.flipkart.varadhi.exceptions;
+
+import lombok.experimental.StandardException;
+
+@StandardException
+public class NotInitializedException extends VaradhiException {
```

Comment:
No need to have this. This type of error falls in IllegalStateExceptions bucket. Something was expected but was not true. It will happen due to a bug and not a common occurrence.